meshnesh/android-level-up
# FT Creating the Retrofit instance #155890684

# What does this PR do?

* Create Java Classes for Resources.
* Creating the Retrofit instance

## How should this be manually tested?

* Clone the repo to your local machine.
* Open the project with Android Studio.
* Check the files on the path: `level_up/service`

## What are the relevant pivotal tracker stories?

[#155889952](https://www.pivotaltracker.com/story/show/155889952)
[#155890684](https://www.pivotaltracker.com/story/show/155890684)
